src/style.css

```diff
@@ -1 +1,81 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+.nav {
+  display: flex;
+}
+
+.nav__list {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  position: absolute;
+  right: 50px;
+  top: 22px;
+}
```

Comment:
Don't position list absolutely, there are easier ways to do it using display flex.